app/javascript/components/open-pr.vue

```diff
@@ -49,7 +70,9 @@
 
 <script>
 import { mapState } from 'vuex';
+import axios from 'axios';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'axios' is defined but never used  no-unused-vars